### DISCUSSION TITLE
Non working display rev1.2 t4.1

### DISCUSSION THREAD

**Discussion by Wizards-of-Zen** [ElectroTechnique/electrotechnique.github.io#48] [Hardware Issues]
Don't think my unit is doing much. Windows 10 sees it in the sound settings. The display is blank
![20210516_131245](https://user-images.githubusercontent.com/84286594/118407642-b0370700-b64f-11eb-9fc4-c942efd9ff61.jpg)

Just loading any firmware

**Comment by Electrotechnique** [comment]
It's a hardware not a firmware problem. Load the latest firmware V.2.20 . Can you hear anything - does it actually work, even if the display doesn't?

**Reply by Wizards-of-Zen** [reply]
Yes the unit passes sound. Thank you for that. The company I bought it from said the display might be bad, sending a replacement

**Reply by Electrotechnique** [reply]
I suspect BLK is soldered to the wrong pin on the Teensy. Can you provide a picture.

Actually, can you just desolder the jumper wire and see if it works?

**Comment by rolfdegen** [comment]
Maybe the resistor on the clock line to the display is too high?
